tests/conftest.py

```diff
@@ -2,10 +2,16 @@
 
 import pytest
 from django.core.cache import cache as default_cache
+from pytest_django.fixtures import SettingsWrapper
 
 from django_redis.cache import BaseCache
 
 
+@pytest.fixture(autouse=True)
+def patch_settings(settings: SettingsWrapper):
```

Comment:
not sure if this is needed everywhere

Reply:
this is probably not needed, I suggest to try to add this default setting in [here](https://github.com/jazzband/django-redis/blob/master/tests/settings/sqlite_herd.py)

Reply:
Done! Moved to `settings_herd.py`.